apc_manager:
  show_goal_bin_markers: true